common/src/main/proto/TransportMessages.proto

```diff
@@ -173,6 +175,14 @@ message PbRegisterShuffleResponse {
   repeated PbPartitionLocation partitionLocations = 2;
 }
 
+message PbHeartbeatFromClient {
+  repeated int32 shuffleId = 1;
```

Comment:
shuffleIds?

Reply:
The generated method name should be `getShuffleIdList`

Reply:
But seems inconsistent with other places in the codebase...

Reply:
ditto